pydoctor/test/test_epydoc2stan.py

```diff
@@ -395,7 +395,41 @@ class C:
     mod.system.intersphinx = InMemoryInventory()
     linker = epydoc2stan._EpydocLinker(mod)
 
-    url = linker.resolve_identifier_xref('socket.socket')
+    url = linker.resolve_identifier_xref('socket.socket', 0)
 
     assert 'https://docs.python.org/3/library/socket.html#socket.socket' == url
     assert not capsys.readouterr().out
+
+
+def test_xref_not_found_epytext(capsys: CapSys) -> None:
```

Comment:
It would be nice to have a docstring explaining the purpose of this test.

```suggestion
def test_xref_not_found_epytext(capsys: CapSys) -> None:
   """
   When reporting an epytext error, the report contains the line number for the reference that triggered the error and not the start line number of the docstring.
   """
```

Reply:
I'll add one, also for the other new test case.